src/hardware/index.ts

```diff
@@ -209,16 +209,26 @@ export const getSignerNameFromType = (type: SignerType, isMock = false, isAmf =
   return name;
 };
 
-export const getWalletConfig = ({ vault }: { vault: Vault }) => {
-  let line = '# Multisig setup file (exported from Keeper)\n';
+export const getWalletConfig = ({ vault, signerType }: { vault: Vault; signerType?: string }) => {
+  const isKeystone = signerType === SignerType.KEYSTONE;
+  let line = '';
+  if (isKeystone) line += 'Y q';
+  line += `# ${isKeystone ? 'Keystone' : ''} Multisig setup file (exported from Keeper)\n`;
```

Comment:
```suggestion
  line += `#${isKeystone ? ' Keystone' : ''} Multisig setup file (exported from Keeper)\n`;
```

It currently has double space if it is not keystone.